index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+
+  <!-- link to fontawesome -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+  <title>Portfolio</title>
+</head>
+<body>
+  <header>
+    <nav class="navbar">
+      <ul>
+        <li class="nav-item logo">Bennyjoez</li>
+        <li class="nav-item logo">
+          <i class="fa-solid fa-bars"></i>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="headline">
+      <h1 class="greeting">
+        Hey There. <br>
+        I'm Benson
+      </h1>
+      <h4 class="title">I'am a Software Developer</h4>
+      <p class="headline-description">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+      </p>
+      <ul class="socials">
+        <li>
+          <a href="https://github.com/Bennyjoez" target="_blank" rel="noopener">
+            <img src="./images/github.svg" alt="github">
+          </a>
+        </li> 
+        <li>
+          <a href="https://www.linkedin.com/in/benson-njuguna-008146242/" target="_blank" rel="noopener">
+            <img src="./images/Linkedinicon.svg" alt="Linkedin">
+          </a>
+        </li> 
+        <li>
+          <a href="#" target="_blank" rel="noopener">
+            <img src="./images/angelist.svg" alt="angelist">
+          </a>
+        </li> 
+        <li>
+          <a href="https://twitter.com/bennyjoezz" target="_blank" rel="noopener">
+            <img src="./images/twitter.svg" alt="twitter">
+          </a>
+        </li> 
+        <li>
+          <a href="#" target="_blank" rel="noopener">
+            <img src="./images/medium.svg" alt="medium">
+          </a>
+        </li> 
+      </ul>
+      <img src="./images/blob1.png" alt="blob" class="blob">
```

Comment:
- Great job for implementing the previous reviewer suggestions :+1: , I think you missed something, you used the section image in the wrong place because the image is not part of the content, If an image is not part of the content (for aesthetic reasons) use a CSS background image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info :+1: .